src/FOHClasses/Waiter.java

```diff
@@ -1,30 +1,31 @@
 package FOHClasses;
 
+import FOHClasses.Collection.OrderCollection;
+import FOHClasses.Collection.PhysicalTableCollection;
+
 public class Waiter {
     private final int staffID;
     private String name;
     private int pin;
     private String phoneNumber;
     private String waiterType;
-    private PhysicalTable tableCurrentlyWaiting;
+    private PhysicalTableCollection tableCurrentlyWaiting;
```

Comment:
For any variable that is holding multiple values (arrays/collections), the variable name should be plural. Example: String[] stringVariables compared to String stringVariable. In this instance, currentTables might be a better variable name. Try to keep variable names short whilst retaining enough information and following the naming convention.